<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Uart Terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 223);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblConnectStat">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>UART TERMINAL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPort">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBaudRate">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Baudrate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbBaudRate">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <stylestrategy>PreferDefault</stylestrategy>
              <hintingpreference>PreferDefaultHinting</hintingpreference>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Temizle</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(38, 162, 105);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <fontweight>ExtraBold</fontweight>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 0px; 

</string>
         </property>
         <property name="text">
          <string>â†µ</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
